main.c

```diff
@@ -527,28 +529,38 @@ STATIC bool run_code_py(safe_mode_t safe_mode, bool first_run, bool *simulate_re
         // Sleep until our next interrupt.
         #if CIRCUITPY_ALARM
         if (result.return_code & PYEXEC_DEEP_SLEEP) {
-            // Make sure we have been awake long enough for USB to connect (enumeration delay).
-            int64_t connecting_delay_ticks = CIRCUITPY_USB_CONNECTED_SLEEP_DELAY * 1024 - port_get_raw_ticks(NULL);
-            // Until it's safe to decide whether we're real/fake sleeping
+            const bool awoke_from_true_deep_sleep =
```

Comment:
Why const?

Reply:
Just indicating I don't expect it to change. If someone wants to change it further down they have to change this and it will make them consider why. Sometimes I use `const` for such safety, and in languages where it's really easy (e.g., `let` vs `var`) I'd consider each one.

I seem to be in a programming-language mood today.